go.mod

```diff
@@ -4,10 +4,7 @@ go 1.18
 
 // runtime dependencies
 require (
-	github.com/Azure/azure-sdk-for-go v65.0.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.11.24
-	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
-	github.com/Azure/go-autorest/autorest/to v0.4.0
+	github.com/Azure/go-autorest/autorest v0.11.24 // indirect
```

Comment:
maybe run a go mod tidy on this? auto rest should be indirect and azidentity should be a direct dependency I guess

Reply:
If I run 'go mod tidy' it deletes everything or does nothing. I don't know why. Could you try that?